### PR TITLE
New query commands

### PR DESCRIPTION
**Pull request by roman-khimov**
I wanted to add a bit more, but we need this now.

### CODE REVIEW

File: cli/query/query.go [hunk 1]
```diff
@@ -113,3 +127,65 @@ func dumpApplicationLog(ctx *cli.Context, res *result.ApplicationLog, tx *result
...
+		if vals[i].Votes != vals[j].Votes {
+			return vals[i].Votes > vals[j].Votes
+		}
+		return vals[i].PublicKey.Cmp(&vals[j].PublicKey) == -1
+	})
+	buf := bytes.NewBuffer(nil)
+	tw := tabwriter.NewWriter(buf, 0, 2, 2, ' ', 0)
+	_, _ = tw.Write([]byte("Key\tVotes\tCommittee\tConsensus\n"))
+	for _, val := range vals {
+		_, _ = tw.Write([]byte(fmt.Sprintf("%s\t%d\t%t\t%t\n", hex.EncodeToString(val.PublicKey.Bytes()), val.Votes, comm.Contains(&val.PublicKey), val.Active)))
```
Comment (AnnaShaleva) [inline cli/query/query.go:174]: Is it possible that candidate does not belong to committee?

File: cli/query/query.go [hunk 2]
```diff
@@ -113,3 +127,65 @@ func dumpApplicationLog(ctx *cli.Context, res *result.ApplicationLog, tx *result
...
+			return vals[i].Active
+		}
+		if vals[i].Votes != vals[j].Votes {
+			return vals[i].Votes > vals[j].Votes
+		}
+		return vals[i].PublicKey.Cmp(&vals[j].PublicKey) == -1
+	})
+	buf := bytes.NewBuffer(nil)
+	tw := tabwriter.NewWriter(buf, 0, 2, 2, ' ', 0)
+	_, _ = tw.Write([]byte("Key\tVotes\tCommittee\tConsensus\n"))
```
Comment (AnnaShaleva) [inline cli/query/query.go:172]: Maybe it would also be useful to print the current block index?

File: docs/consensus.md [hunk 3]
```diff
@@ -92,16 +92,16 @@ use.
 This command will create and send appropriate transaction to the network and
 you should then wait for it to settle in a block. If all goes well it'll end
 with "HALT" state and your registration will be completed. You can use
-`query tx` command to see transaction status or
-`getnextblockvalidators` to see if your candidate was added.
+`query tx` command to see transaction status or `query candidates` to see if
```
Comment (AnnaShaleva) [inline docs/consensus.md:95]: A couple of words about `query committee`?

File: cli/query/query.go (hunk 1 above)
Comment (roman-khimov) [reply cli/query/query.go:174]: We can have like 50 candidates with 21 committee members.

File: cli/query/query.go (hunk 2 above)
Comment (roman-khimov) [reply cli/query/query.go:172]: Not sure about that, it doesn't fit naturally there.

File: docs/consensus.md (hunk 3 above)
Comment (roman-khimov) [reply docs/consensus.md:95]: It doesn't make sense here, but you're right in that cli.md should be updated.

Comment (roman-khimov) [reply docs/consensus.md:95]: Fixed.

File: cli/query/query.go [hunk 4]
```diff
@@ -189,3 +199,59 @@ func queryCommittee(ctx *cli.Context) error {
...
 	return nil
 }
+
+func queryVoter(ctx *cli.Context) error {
+	args := ctx.Args()
+	if len(args) == 0 {
+		return cli.NewExitError("No address specified", 1)
+	}
+
+	addr, err := flags.ParseAddress(args[0])
```
Comment (AnnaShaleva) [inline cli/query/query.go:243]: Don't we want to add an address flag instead of an argument?

File: cli/query_test.go [hunk 5]
```diff
@@ -130,3 +130,23 @@ func (e *executor) compareQueryTxVerbose(t *testing.T, tx *transaction.Transacti
...
+	/*
+		height := e.Chain.BlockHeight()
+		go e.Chain.Run()
+		require.Eventually(t, func() bool { return e.Chain.BlockHeight() > height }, time.Second*2, time.Millisecond*50)
+
+		e.Run(t, "neo-go", "query", "height", "--rpc-endpoint", "http://"+e.RPC.Addr)
+		e.checkNextLine(t, `Latest block: \d+\s+`)
+		e.checkNextLine(t, `Validated state: 0\s+`)
+		e.checkEOF(t)
+	*/
```
Comment (AnnaShaleva) [inline cli/query_test.go:151]: Do we need this code?

**Review by AnnaShaleva — CHANGES_REQUESTED** [review]

File: cli/query/query.go (hunk 4 above)
Comment (roman-khimov) [reply cli/query/query.go:243]: It's mandatory, so why bother with flags? I can't remember all flags, I just want to `query smth for_this_address`.

File: cli/query_test.go (hunk 5 above)
Comment (roman-khimov) [reply cli/query_test.go]: Meh, slipped through. Fixed.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**